proc_nutation.py

```diff
@@ -33,13 +34,14 @@
         searchstr,
         exp_type=exp_type,
         expno=nodename,
-        postproc=postproc,
+        postproc=postproc,# PR why do you need to manually specify the postproc? that should not be needed!
         lookup=lookup_table,
         fl=fl,
     )
     s = s["t2":freq_slice]
     fl.next("data")
     fl.image(s)
+    # {{{ PR why are we doing the following -- is it needed?? If it is, comment on this, and also include a slide in your pptx
```

Comment:
deal with this (applies to entire bracketed region)

Reply:
again this was an old old version not in the examples repo and has since been deleted. The up to date version is inside the examples directory